README.md

```diff
@@ -11,6 +11,22 @@
 
 ConfigCat SDK for JavaScript provides easy integration for your application to [ConfigCat](https://configcat.com).
 
+This repository hosts the next generation of the ConfigCat SDK for JavaScript platforms. It ships in the form of a single, unified NPM package for
+different JS platforms, as opposed to the one package per platform model used before.
+
+The new SDK combines and, thus, supersedes these packages:
+* [configcat-common](https://www.npmjs.com/package/configcat-common)
+* [configcat-js](https://www.npmjs.com/package/configcat-js)
+* [configcat-js-ssr](https://www.npmjs.com/package/configcat-js-ssr)
+* [configcat-js-chromium-extension](https://www.npmjs.com/package/configcat-js-chromium-extension)
+* [configcat-node](https://www.npmjs.com/package/configcat-node)
+
+The new SDK maintains backward compatibility, so it can be used as a drop-in replacement for the packages listed above. In most cases you just need to
+replace the old package with the new and adjust the import specifiers (as shown [here](#1-install-and-import-package)).
+
+> [!CAUTION]
+> Please note that the SDK is still under development and is currently in beta phase. You use it at your own risk.
```

Comment:
The "You use it at your own risk." part maybe not necessary, it sounds a bit harsh to me.